tests/realtikvtest/pipelineddmltest/pipelineddml_test.go

```diff
@@ -740,6 +741,7 @@ func TestConflictError(t *testing.T) {
 }
 
 func TestRejectUnsupportedTables(t *testing.T) {
+	t.Fail() // Does this package really tested?
```

Comment:
Seems this package is not tested, need to figure out why.
![image](https://github.com/pingcap/tidb/assets/9587680/81e30d53-2073-4af9-8ded-578639242a29)